samples/WOCCatalog/WOCCatalog/URLSessionViewController.mm

```diff
@@ -107,19 +113,28 @@ - (void)loadView {
     [self.view addSubview:_blockSwitchContainer];
 
     UIButton* cancelLastButton = [UIButton buttonWithType:UIButtonTypeRoundedRect];
-    cancelLastButton.frame = (CGRect){ BUTTON_ROW_ITEM_X(nbuttons % BUTTON_NUM_IN_ROW), BUTTON_ROW_ITEM_Y(nbuttons++ / BUTTON_NUM_IN_ROW), BUTTON_ROW_ITEM_WIDTH, BUTTON_ROW_HEIGHT };
+    cancelLastButton.frame = (CGRect){ BUTTON_ROW_ITEM_X(nbuttons % BUTTON_NUM_IN_ROW),
+                                       BUTTON_ROW_ITEM_Y(nbuttons++ / BUTTON_NUM_IN_ROW),
```

Comment:
Oops, looks like the rules changed in C++11 so that the ordering is now well-defined. Feel free to ignore me...

---
In reply to: [70360966](https://github.com/Microsoft/WinObjC/pull/632#discussion_r70360966) [](ancestors = 70360966)